### PR TITLE
Constant model created along with test files and example notebook

### PR DESCRIPTION
**Pull request by ben18785**

### CODE REVIEW

File: pints/toy/_constant_model.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import numpy as np
+import pints
+
+
+class ConstantModel(pints.ForwardModel):
+    """
+    *Extends:* :class:`pints.ForwardModel`.
+    .. math::
+        f(t) &= (a_1,a_2,...,a_k)
+
```
Comment (MichaelClerx) [inline pints/toy/_constant_model.py]: I don't get this description!

File: pints/toy/_constant_model.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    This function is mostly useful for unit testing.
+
+    """
+
+    def __init__(self):
+        super(ConstantModel, self).__init__()
+        self._no = -99
+        self._parameters = -99
+
+    def parameters(self):
```
Comment (MichaelClerx) [inline pints/toy/_constant_model.py]: n_parameters

File: pints/toy/_constant_model.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+    Has a vector of parameters of dimensionality k: each of which
+    is the user-specified output of a given component of the function.
+    This function is mostly useful for unit testing.
+
+    """
+
+    def __init__(self):
+        super(ConstantModel, self).__init__()
+        self._no = -99
```
Comment (MichaelClerx) [inline pints/toy/_constant_model.py]: What's going on here??

Comment (MichaelClerx) [reply pints/toy/_constant_model.py]: Maybe let users specify n_parameters and n_outputs in the constructor?

**Comment by ben18785** [comment]
Yep, agreed. I’ll make those changes; good idea.

This model is useful for testing likelihoods — hopefully I can make this clear in my revamped tests.

**Pull request closed without merging**